ao_mrp_flattened_bom_cost_xlsx/__manifest__.py

```diff
@@ -7,9 +7,9 @@
     'version': '11.0.1.0.0',
     'category': 'Manufacturing',
     'author': "Eficent",
-    'website': 'https://github.com/Eficent',
+    'website': 'https:www,eficent.com',
```

Comment:
the comma 😬

Reply:
and also needs the two `//`

Reply:
This will be a v12 improvement :rofl: :rofl: :rofl: